pkg/reconciler/inmemorychannel/controller/resources/dispatcher.go

```diff
@@ -69,12 +69,12 @@ func MakeDispatcher(args DispatcherArgs) *v1.Deployment {
 							Image: args.Image,
 							Env:   makeEnv(),
 
-							// Set resource requests and limits based on the benchmarking results in
-							//  https://github.com/knative/eventing/issues/2311#issuecomment-565311345
+							// Set low resource requests and limits.
+							// This should be configurable.
```

Comment:
Create an issue and add a TODO?

Reply:
we have many already :-) For instance: https://github.com/knative/eventing/issues/923